protocol/communities/community.go

```diff
@@ -1548,15 +1548,14 @@ func (o *Community) tokenPermissions() map[string]*CommunityTokenPermission {
 func (o *Community) PendingAndBannedMembers() map[string]CommunityMemberState {
 	result := make(map[string]CommunityMemberState)
 
-	// Non-privileged members should not see pending and banned members
```

Comment:
I return all updated data. If not to do it, on the client there will be unexpected behavior:
1. Before adding `PendingAndBannedMembers` fn, we returned `BanList` and didn't filter anything on go side 
2. A client on a startup has a ban list received by `ChannelGroup` struct 
3. Client must receive updates, when he's added to the ban list and when he's moved from the ban list (now we have a bug)